src/generated/client/models/User.ts

```diff
@@ -39,10 +39,10 @@ export interface User {
      */
     name: string;
     /**
-     * User email
-     * @type {string}
-     * @memberof User
-     */
+      * User name
```

Comment:
```suggestion
      * User email
```
It shouldn't be changed to name if it says email. If email returns name - the error should be fixed from it's source